README.md

```diff
@@ -30,6 +30,12 @@ or
 ninja full.img.gz
 ```
 
+In case of use domd rootfs from the usb flash drive rootfs partition will not be created in image. Rootfs
+image should be written separately to the appropriate partition on media with command:
+```
+dd if=yocto/build-domd/tmp/deploy/images/raspberrypi5/rpi5-image-xt-domd-raspberrypi5.rootfs.ext4 of=<partition device> bs=1M
+```
+Note: before writing rootfs media should be properly partitioned for example with `fdisk` command
```

Comment:
Well, this might work. But I believe it would be easier for user if you just write additional `image` section in YAML to create an image for USB stick.

Reply:
I have created additional `image`